.travis.yml

```diff
@@ -26,7 +26,8 @@ env:
 before_install:
   # See https://github.com/travis-ci/travis-ci/issues/4629#issuecomment-239493916
   - rm ~/.m2/settings.xml
-  - curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
+  - if [ ! -f ~/.jabba/jabba.sh ]; then curl -L -v --retry 5 -o jabba-install.sh https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh && bash jabba-install.sh; fi
+  - . ~/.jabba/jabba.sh
```

Comment:
After several issues in travis trying to fetch the contents of git.io/jabba.sh (I think there's a dirty proxy cache on travis network) I decided to get some code from https://github.com/playframework/playframework/blob/master/.travis.yml#L32 and also use the static URL.

Reply:
Pfff! Even using the static URL (raw.githubusercontent.com) Travis still produces HTML:

https://travis-ci.com/github/lagom/lagom/jobs/497110725#L230-L357

I think I'll have to wait for tavis to sort this one out (expire their caches).

Reply:
Time to go to GitHub Actions. 😄

Reply:
> Time to go to GitHub Actions. 😄
Are you volunteering? 😂

Reply:
> Are you volunteering? 😂

Well, give it a 30 minutes try: https://github.com/lagom/lagom/pull/3216.

It is incomplete, but it is a good start. @ihostage if you have the time, feel free to take over that PR. :-)